contents/tutorials/performance-metrics.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Understanding page load time metrics in performance monitoring
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-12
+featuredImage: todo
+tags: []
+---
+
+Waiting for slow websites is like watching paint dry. It's the bane of productivity, the destroyer of efficiency, and the enemy of progress.
+
+In this tutorial, we'll show you how to understand different metrics on how quickly your pages load and how to identify opportunities for improvement using Posthog's [performance monitoring in session recordings tool](https://posthog.com/blog/posthog-changelog#performance-monitoring-in-session-recordings).
+
+## Setup
+
+First, ensure that session recordings are [enabled]((https://posthog.com/docs/session-replay/manual)). Next, navigate to the Recordings tab on your Posthog Dashboard. Finally, click on the Network tab in the sidebar next to or below the video.
+
+![](https://i.imgur.com/nr0QSSy.png)
+
+Here you'll see all the network requests that are made during the page load and user session, as well as how long they took. You'll also see the time taken for First Contentful Paint, DOM interactive, and Page Loaded. 
+
+![](https://i.imgur.com/Qlqt4c0.png)
+
+We'll go through what each one of these three metrics mean and how to improve them.
+
+## First Contentful Paint
```

Comment:
```suggestion
## 1. First contentful paint
```

Reply:
The convention seems to be to use `First Contentful Paint` since it's a proper noun (although I can also find example of "First contentful paint")

https://www.google.com/search?q=first+contentful+paint&oq=first+con&aqs=chrome.0.69i59j69i57j69i59l2j35i39j69i60l2j69i61.1768j0j7&sourceid=chrome&ie=UTF-8